src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -26,21 +26,20 @@
     QuickBuildRequest,
     PlatformProperties
 )
-from ._utils import get_resource_group_name_by_registry_name
+from ._utils import validate_managed_registry
 from ._client_factory import cf_acr_registries
 
+
 logger = get_logger(__name__)
 
 
-def acr_build_show_logs(cmd,
-                        client,
-                        registry_name,
-                        build_id,
-                        resource_group_name=None):
+BUILD_NOT_SUPPORTED = 'Builds are only supported for managed registries.'
 
-    resource_group_name = get_resource_group_name_by_registry_name(
-        cmd.cli_ctx, registry_name, resource_group_name)
 
+def acr_build_show_logs(client,
+                        build_id,
+                        registry_name,
+                        resource_group_name):
```

Comment:
Why did we make `resource_group_name` required argument?

Reply:
This is an internal method and the caller already has `resource_group_name` resolved. Unnecessary overhead here.